app/models/piece.rb

```diff
@@ -18,12 +17,53 @@ def move!(x, y)
     true
   end
 
+  # First check if coordinates is taken
+  # return false if coordinates belongs to friendly piece
+  # otherwise return true
+  # return false if position is not taken
+  def capturable?(x, y)
+    if position_taken?(x, y)
+      return false if friendly_piece?(x, y)
+      true
+    else
+      false
+    end
+  end
```

Comment:
This whole method can be refactored into `position_taken(x, y) && ! friendly_piece?(x, y)`.  Much simpler.